docs/io/votable/index.rst

```diff
@@ -261,26 +260,40 @@ input.
 
 .. _pedantic-mode:
 
-Pedantic mode
-^^^^^^^^^^^^^
+Verifying VOTables
+^^^^^^^^^^^^^^^^^^
 
-Many VOTable files in the wild do not conform to the VOTable
-specification. If reading one of these files causes exceptions, you
-may turn off pedantic mode in `astropy.io.votable` by passing
-``pedantic=False`` to the `~astropy.io.votable.parse` or
-`~astropy.io.votable.parse_single_table` functions::
+Many VOTable files in the wild do not conform to the VOTable specification. You
+can set what should happen when a violation is encountered with the ``verify``
+argument, which can take three values:
+
+    * ``'ignore'`` - Attempt to parse the VOTable silently. This is the default
+      setting.
+    * ``'warn'`` - Attempt to parse the VOTable, but raise appropriate
+      :ref:`warnings`. It is possible to limit the number of warnings of the
+      same type to a maximum value using the
+      `astropy.io.votable.exceptions.conf.max_warnings
+      <astropy.io.votable.exceptions.Conf.max_warnings>` item in the
+      :ref:`astropy_config`
+    * ``'exception'`` - Do not parse the VOTable and raise an exception.
+
+The ``verify`` argument can be used with the :func:`~astropy.io.votable.parse` or
+:func:`~astropy.io.votable.parse_single_table` functions::
 
   from astropy.io.votable import parse
-  votable = parse("votable.xml", pedantic=False)
+  votable = parse("votable.xml", verify='warn')
+
+It is possible to change the default ``verify`` value through the
+`astropy.io.votable.conf.verify <astropy.io.votable.Conf.verify>` item in the
+:ref:`astropy_config`.
 
-Note, however, that it is good practice to report these errors to the
-author of the application that generated the VOTable file to bring the
-file into compliance with the specification.
+Note that ``'ignore'`` or ``'warn'``  mean that ``astropy`` will attempt to
+parse the VOTable, but if the specification has been violated then success
+cannot be guaranteed.
 
-Even with ``pedantic`` turned off, many warnings may still be omitted.
-These warnings are all of the type
-`~astropy.io.votable.exceptions.VOTableSpecWarning` and can be turned
```

Comment:
I see that this sentence is deleted altogether. Does this no longer apply?

Reply:
This was explaining that `pedantic=False` can emit (not omit, that's a typo) warnings, which might have been unexpected. What the `verify` options do is already explained above.